Workbooks/ManualSentinelIncident.json

```diff
@@ -0,0 +1,301 @@
+                  "typeSettings": {
+                    "additionalResourceOptions": []
+                  },
+                  "queryType": 8,
+                  "value": "High"
+                },
+                {
+                  "id": "607f84eb-4528-4763-8ca9-d4900c669b02",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "status",
+                  "label": "Status",
+                  "type": 2,
+                  "isRequired": true,
+                  "query": "{\"version\":\"1.0.0\",\"content\":\"[\\\"New\\\", \\\"Active\\\", \\\"Closed\\\"]\",\"transformers\":null}",
+                  "typeSettings": {
+                    "additionalResourceOptions": []
+                  },
+                  "queryType": 8,
+                  "value": "New"
+                },
+                {
+                  "id": "0163c733-ee70-4061-92ce-6ed1761a038d",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "UserUPN",
+                  "label": "UserPrincipalName",
+                  "type": 1,
+                  "description": "Fill in your UPN",
+                  "isRequired": true,
+                  "value": ""
+                },
+                {
+                  "id": "580ed3d7-6a63-4a53-95c2-2e127e0b51d4",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "GUID",
+                  "label": "ID for Sentinel Incident",
+                  "type": 1,
+                  "isRequired": true,
+                  "query": "print guid=new_guid()",
+                  "crossComponentResources": [
+                    "{Workspace}"
+                  ],
+                  "typeSettings": {
+                    "paramValidationRules": [
+                      {
+                        "regExp": "^{?([0-9a-fA-F]){8}(-([0-9a-fA-F]){4}){3}-([0-9a-fA-F]){12}}?$",
+                        "match": true,
+                        "message": ""
+                      }
+                    ]
+                  },
+                  "timeContext": {
+                    "durationMs": 86400000
+                  },
+                  "queryType": 0,
+                  "resourceType": "microsoft.operationalinsights/workspaces"
+                }
+              ],
+              "style": "formVertical",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "name": "parameters - 4"
+          },
+          {
+            "type": 11,
+            "content": {
+              "version": "LinkItem/1.0",
+              "style": "list",
+              "links": [
+                {
+                  "id": "0ce7d423-77f7-4221-b79b-787164bb4935",
+                  "linkTarget": "ArmAction",
+                  "linkLabel": "Create Incident",
+                  "preText": "❗",
```

Comment:
Hello @pthoor The validation check might be failing due to "pretext": "!"
Is this string valid or should be changed